people.md

```diff
@@ -0,0 +1,60 @@
+# People in the governance structure
+
+This document lists the current people and institutions involved in
+the governance.
+
+Benevolent Dictator for Life
+============================
+
+Fernando Perez is the Benevolent Dictator for Life (BDFL).
+
+Steering Council
+================
+
```

Comment:
+1 to alphabetical as per `M-x sort-lines` for simplicity, we could just add in parens a year of joining, which could contain a year of leaving for folks who leave.